internal/tools/securitypolicy/main.go

```diff
@@ -192,48 +195,43 @@ func createPolicyFromConfig(config Config) (sp.SecurityPolicy, error) {
 			}
 			hash := dmverity.RootHash(tree)
 			hashString := fmt.Sprintf("%x", hash)
-			container.Layers = addLayer(container.Layers, hashString)
+			addLayer(&container.Layers, hashString)
```

Comment:
I might be forgetting something, but did we discuss `addLayer`, `addEnvRul`, `addContainer` potentially being a function of `securitypolicy.SecurityPolicy` struct?

Reply:
We didnt discuss that, but we did discuss some reorganization for this tool that is coming in a later change.

Reply:
are you saying it doesn't make sense to make the change I'm suggesting, because the code is going to be reorganized?

Reply:
im saying it is part of a different set of changes,

Reply:
yeah, ok. we just want to have smaller patches. makes sense.